### PR TITLE
ddl: speed up tests

### PR DESCRIPTION
**Pull request by zimulala**
### What problem does this PR solve? 
Speed up testing time.

### What is changed and how it works?

- TestDropColumn tests `drop column`, so I will add the required columns when I create the table.

- Change multiple insert statements to one.

```
go test  -v -vet=off -timeout 10m  -race -count 1
```
Before this PR:
```
Local
ok      github.com/pingcap/tidb/ddl    78.583s

CI
2m 23s(https://internal.pingcap.net/idc-jenkins/blue/organizations/jenkins/tidb_ghpr_unit_test/detail/tidb_ghpr_unit_test/11707/pipeline/75/) 
```
After this PR:
```
Local
ok      github.com/pingcap/tidb/ddl    67.245s

CI
1m 38s(https://internal.pingcap.net/idc-jenkins/blue/organizations/jenkins/tidb_ghpr_unit_test/detail/tidb_ghpr_unit_test/11811/pipeline/75)
```

### CODE REVIEW

File: ddl/failtest/fail_db_test.go [hunk 1]
```diff
@@ -336,10 +336,14 @@ func (s *testFailDBSuite) TestAddIndexWorkerNum(c *C) {
 	start := -10
 	num := 4096
 	// first add some rows
+	dml := "insert into t1 test_add_index"
```
Comment (bb7133) [inline ddl/failtest/fail_db_test.go]:
```suggestion
	dml := "insert into t1 test_add_index values"
```
Please fix this typo

**Comment by zimulala** [comment]
@bb7133 I think it is enough to change two. Most of the other tests are internal tests, not using `MustExec`.

File: ddl/db_test.go [hunk 2]
```diff
@@ -225,6 +226,17 @@ func backgroundExec(s kv.Storage, sql string, done chan error) {
 	done <- errors.Trace(err)
 }
 
+func batchInsert(tk *testkit.TestKit, tbl string, start, end int) {
+	dml := fmt.Sprintf("insert into %s values", tbl)
```
Comment (djshow832) [inline ddl/db_test.go]: using `bytes.Buffer` in concat strings should be more efficient.

File: ddl/failtest/fail_db_test.go [hunk 3]
```diff
@@ -334,11 +345,9 @@ func (s *testFailDBSuite) TestAddIndexWorkerNum(c *C) {
 
 	done := make(chan error, 1)
 	start := -10
-	num := 4096
 	// first add some rows
-	for i := start; i < num; i++ {
-		sql := fmt.Sprintf("insert into test_add_index values (%d, %d, %d)", i, i, i)
-		tk.MustExec(sql)
+	for i := start; i < 4090; i += 100 {
```
Comment (djshow832) [inline ddl/failtest/fail_db_test.go]: Actually 4100 records are inserted, does it matter?

File: ddl/db_test.go [hunk 4]
```diff
@@ -1519,15 +1527,22 @@ func (s *testDBSuite2) TestDropColumn(c *C) {
...
+		multiDDL = append(multiDDL, fmt.Sprintf("alter table t2 drop column c%d", i))
+
+		if i != 3+num {
+			sql += fmt.Sprintf("c%d int, ", i)
+		} else {
+			sql += fmt.Sprintf("c%d int)", i)
+		}
+	}
+	s.tk.MustExec(sql)
 	dmlDone := make(chan error, num)
```
Comment (djshow832) [inline ddl/db_test.go]: `dmlDone` is never checked, so it doesn't mean c4 is actually dropped.

File: ddl/db_test.go [hunk 5]
```diff
@@ -1519,15 +1527,22 @@ func (s *testDBSuite2) TestDropColumn(c *C) {
...
 	ddlDone := make(chan error, num)
 
-	multiDDL := make([]string, 0, num)
-	for i := 0; i < num/2; i++ {
-		multiDDL = append(multiDDL, "alter table t2 add column c4 int", "alter table t2 drop column c4")
-	}
 	testddlutil.ExecMultiSQLInGoroutine(c, s.store, "drop_col_db", multiDDL, ddlDone)
 	for i := 0; i < num; i++ {
 		testddlutil.ExecMultiSQLInGoroutine(c, s.store, "drop_col_db", []string{"insert into t2 set c1 = 1, c2 = 1, c3 = 1, c4 = 1"}, dmlDone)
```
Comment (djshow832) [inline ddl/db_test.go]: If you add c5~c28, those columns should also be checked through INSERT.

File: ddl/failtest/fail_db_test.go (hunk 3 above)
Comment (zimulala) [reply ddl/failtest/fail_db_test.go]: `start` is -10, so I think 4090 is OK.

File: ddl/db_test.go [hunk 6]
```diff
@@ -1519,15 +1527,22 @@ func (s *testDBSuite2) TestDropColumn(c *C) {
 	s.tk = testkit.NewTestKit(c, s.store)
 	s.tk.MustExec("create database drop_col_db")
 	s.tk.MustExec("use drop_col_db")
-	s.tk.MustExec("create table t2 (c1 int, c2 int, c3 int)")
-	num := 50
+	num := 25
+	multiDDL := make([]string, 0, num)
+	sql := "create table t2 (c1 int, c2 int, c3 int, "
+	for i := 4; i < 4+num; i++ {
+		multiDDL = append(multiDDL, fmt.Sprintf("alter table t2 drop column c%d", i))
```
Comment (AilinKid) [inline ddl/db_test.go]: that means dropping the column sequentially, maybe more disorderedly.

**Comment by zimulala** [comment]
PTAL @djshow832

**Pull request merged**